tests/Feature/Commands/TweetAboutLiveStreamsCommandTest.php

```diff
@@ -29,6 +30,20 @@ public function it_only_tweets_streams_that_are_live(): void
         $this->twitterFake->assertTweetWasSent();
     }
 
+    /** @test */
+    public function it_does_not_tweet_streams_that_are_live_or_finished(): void
```

Comment:
@christophrumpel Shouldn't this be `it_does_not_tweet_streams_that_are_upcoming_or_finished` ?